### DISCUSSION TITLE
???-Module 7: Open Source GIS Web Development-???

### DISCUSSION THREAD

**Discussion by aaronbirr** [ua-gist604b-f24/syllabus#44] [General]
I believe I may have overlooked some instructions. Are we no longer proceeding with Module 7?

**Comment by aaronbirr** [comment]
I also noticed that Module 8 in D2L is listed as 50 points. Could someone confirm if this is correct?

8-0 GDAL | - / 50 |

**Reply by jenw-ua** [reply]
I feel like at the beginning of the class, in one of the videos, he said every assignment is worth 10 points. So I think that's just something that will be updated.  As for Module 7, I hope we will have more than Monday and Tuesday to work on it? It's Sunday and the last few weeks, he seems to post videos, etc on Sunday afternoons, so I'm hoping we see something soon. 

...as far as I know, you have not overlooked anything. I'm definitely in the same boat as you, waiting to see what's next.

**Comment by jimmy-nguyen-data-science** [comment]
Yeah, what's going on? Are we just doing the module 8 assignment without the module 7 assignments? There are no lectures posted or anything either I've been waiting for further instructions but now I feel like I just do module 8 instead.

**Reply by aaryno** [reply]
I'm going to rearrange - I need to test it again before I release it. Look for those later today with a different due date.

**Comment by aaronbirr** [comment]
Thanks for the responses, I noticed others going onto Module 8 so I'm going to move onto that, and wait and see about 7. But certainly share your concerns.
